### ISSUE TITLE
Fab jumps around

### ISSUE THREAD

**Issue by JereIsThere** [Threatening-Code-Comments/RoutineTimer#1]
Fab in selectroutine jumps around - probably due to specifying layout-gravity on the fab

**Comment by Yoshie2000** [comment]
NO, it's because you're using a recyclerview in a collapsing toolbarlayout. maybe SOMEONE should go change that.

**Comment by JereIsThere** [comment]
You're right, but right now this is a very minor problem compared to the bunch of shit that needs to be done still

**Comment by Yoshie2000** [comment]
‏‏‎ ‎
